dask_kubernetes/tests/conftest.py

```diff
@@ -1,13 +1,52 @@
+import asyncio
+
 import pytest
 
+import kubernetes
+
+from dask_kubernetes.auth import ClusterAuth, KubeConfig, InCluster
+
 
 def pytest_addoption(parser):
-    parser.addoption("--worker-image", help="Worker image to use for testing")
+    parser.addoption(
+        "--worker-image",
+        default="daskdev/dask:dev",
```

Comment:
This should be `daskdev/dask:latest`. And really it should be a custom image which installs `dask` and `distributed` from master.

Reply:
Good point, there is a mismatch between worker and "tester" image. I will address that - in fact, with layer sharing on the `kind` node it should be faster too